Project/Assets/Prairie/Framework/Script/Annotation/AreaAnnotationGui.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.UI;
+using UnityEngine;
+
+public class AreaAnnotationGui : MonoBehaviour
+{
+	private bool active = false;
+
+	public Transform contentPanel;
+	public GameObject prefab;
+	public FirstPersonInteractor FPI;
+
+	// Name of the areaAnnotationGUI object in game scene.
+	private const string AREAANNOTATION = "AnnotationPanel";
+
+	void Start() {
+		gameObject.transform.Find(AREAANNOTATION).gameObject.SetActive (active);
+	}
+
+	public void ActivateGui() {
```

Comment:
Could you maybe add some comments on these small methods just detailing where and when they might be called? I can't tell just from looking at this code.